pivotal-cf/pks-kubernetes-release
# Enable etcd peer verify

**What this PR does / why we need it**:

Since bosh-dns feature is in, we need remove the skip-peer-check flags of etcd. 
This should be merge together with https://github.com/pivotal-cf/kubo-service-adapter-release/pull/237 

**How can this PR be verified?**
https://pks.ci.cf-app.com/teams/bosh-lifecycle-dev/pipelines/pks-api-1.10.x-enable-etcd-peer-verify
**Is there any change in kubo-deployment?**

**Is there any change in kubo-ci?**
No
**Does this affect upgrade, or is there any migration required?**
No
**Which issue(s) this PR fixes:**
https://jira.eng.vmware.com/browse/PKS-3506
**Release note**:

```release-note

```
